.github/workflows/deploy-rtrans.yml

```diff
@@ -66,6 +66,15 @@ jobs:
           insecure_skip_tls_verify: true
           namespace: ${{ env.DEFAULT_NAMESPACE }}
 
+      # Validate that a production deployment uses a release built from the main branch
+      - name: Validate that a production deployment uses a release built from the main branch
+        uses: actions/github-script@v4
+        with:
+          script: |
+            if (`${{ github.event.inputs.Environment }}` == 'production' && `${{ github.event.inputs.Branch }}` != 'main') {
+              core.setFailed("Production deployment can only be built from the main branch. Please build the release from main branch and use that build." );
+            }
+
```

Comment:
Don't think we need this because, VERSION_PREFIX is fixed to "release_". So deploy workflow will only use images with name release_1.0.